src/main/java/org/jabref/gui/entryeditor/CommentsTab.java

```diff
@@ -140,7 +143,8 @@ protected void setupPanel(BibEntry entry, boolean compressed) {
             boolean isStandardBibtexComment = field == StandardField.COMMENT;
             boolean isDefaultOwnerComment = field.equals(userSpecificCommentField);
             boolean shouldBeEnabled = isStandardBibtexComment || isDefaultOwnerComment;
-            editor.getNode().setDisable(!shouldBeEnabled);
+            ((MarkdownEditor)editor.getNode()).setEditable(shouldBeEnabled);
```

Comment:
This is wrong. The text should not be editable, but only selectable.

Reply:
I think their requirement is that users should be able to edit their own comments, and other users' comments should be selectable (copyable). The current code can implement this functionality.